### ISSUE TITLE
Concept Art: Crossbow Weapon

### ISSUE THREAD

**Issue by garrynewman** [Facepunch/rust-issues#101]

**Comment by Megan-T** [comment]
Trying out a few ideas for the crossbow: 
![crossbows](https://cloud.githubusercontent.com/assets/11884607/7140127/ace597aa-e2c1-11e4-9729-40185c37c313.jpg)

**Comment by Helkus** [comment]
Lets do some polished versions of #3 looks great!

**Comment by y0himba** [comment]
I am partial to #4.  Not that it matters, but it looks more primitive and hand constructed. 3 is just too well manufactured.
4 looks more like a primitive arbalest with a handle and trigger, the butt and stock are one piece and it is plausible

Suggestion: What about this:

![xbow_parts](https://cloud.githubusercontent.com/assets/11462603/7146068/e8c7e4a6-e2be-11e4-8716-a25ed071c0a9.png)

**Comment by Gooseman1977** [comment]
They all look wonderful but I'm gonna vote for 4.

**Comment by OverlordJ19** [comment]
I agree @y0himba, however i also think that #1 is a possibility, all the guns have handles and pistol grip styles, I prefer the idea of a basic, back to its roots crossbow. Same as in your picture @y0himba, a single piece of wood made into a crossbow seems nice.

~300 wood, 75 metal frags, 75 cloth???
What u guys think?

**Comment by SnarlyBus** [comment]
i think we should have two eventually. first start with a "rusty" looking one like #4 then u can get one thats a bit better, more like #1. same with bows, i already know u guys got concepts for the recurve bows which look pretty good, but a bit too much like farcry 3

**Comment by ChazFales** [comment]
They all do look great. I really like #3 looks most like something you could make in the wilderness, but I like how simple the one y0himba posted. but keep up the great work.
